vault/logical_system.go

```diff
@@ -3291,6 +3293,14 @@ func (b *SystemBackend) pathHashWrite(ctx context.Context, req *logical.Request,
 		hf = sha512.New384()
 	case "sha2-512":
 		hf = sha512.New()
+	case "sha3-224":
```

Comment:
What about updating the documentation for this operation in `vault/logical_system_paths.go`?